app/Livewire/Project/Service/Storage.php

```
@@ -39,30 +67,151 @@ public function refreshStoragesFromEvent()
+        $this->name = '';
+        $this->mount_path = '';
+        $this->host_path = null;
+        $this->file_storage_path = '';
+        $this->file_storage_content = null;
+        $this->file_storage_directory_destination = '';
+
+        if (str($this->resource->getMorphClass())->contains('Standalone')) {
+            $this->file_storage_directory_source = database_configuration_dir()."/{$this->resource->uuid}";
+        } else {
+            $this->file_storage_directory_source = application_configuration_dir()."/{$this->resource->uuid}";
+        }
```

PHPMD (2.15.0)

The method clearForm uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)